knora/dsplib/schemas/data.xsd

```diff
@@ -454,12 +475,12 @@
         <xs:complexType>
             <xs:sequence>
                 <xs:element name="permissions" type="permissions_type" minOccurs="0" maxOccurs="unbounded"/>
-                <xs:element name="resource" type="resource_type" minOccurs="1" maxOccurs="unbounded">
-                    <xs:unique name="UniqueNameAttr">
-                        <xs:selector xpath=".//*"/>
-                        <xs:field xpath="@name"/>
-                    </xs:unique>
```

Comment:
I didn't succeed in incorporating this <xs:unique> into my new structure. It doesn't seem to be necessary. I also cannot understand what it means. So I wanted to ask if someone of you understands this <xs:unique> and think it should remain.